packages/css/src/components/form/form-group.scss

```diff
@@ -142,14 +147,36 @@
 
   > .dso-label-container {
     @include make-sm-column($dso-label-column);
+
+    @media screen and (min-width: $screen-sm-min) {
+      padding-top: $u1;
+    }
+
+    .control-label {
+      display: inline;
```

Comment:
Waarom is deze nodig, hierboven zet je `.control-label` al op `display: inline;`

Reply:
Klopt, deze lijkt nu wel dubbel te zijn. Ik haal hem weg